microsoft-365/bookings/bookings-in-outlook.md

```diff
@@ -57,17 +57,25 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
 
 1. Bookings with me and Bookings share the same licensing model. However, Bookings doesn't have to be turned on for the organization using tenant settings for users to access Bookings with me. The Bookings app must be enabled for users to have access to Bookings with me.
 
-   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy?view=exchange-ps) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
+   To turn on Bookings with me without access to Bookings, block access to Microsoft Bookings using the [OWA Mailbox policy PowerShell command](/powershell/module/exchange/set-owamailboxpolicy) or follow the instructions here: [Turn Microsoft Bookings on or off](turn-bookings-on-or-off.md).
 
 2. Calendar FreeBusy Anonymous sharing must be enabled to use Bookings with me. This allows the Bookings page to have access to the free/busy information in your Outlook calendar. Use PowerShell to check the status.
 
    ```PowerShell
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
+You should have one of the following domains in the response:
 
-   To enable anonymous sharing, use the following command.
+    Anonymous:CalendarSharingFreeBusySimple: Share free/busy hours only.
+    
+    Anonymous:CalendarSharingFreeBusyDetail: Share free/busy hours, subject, and location.
+    
+    Anonymous:CalendarSharingFreeBusyReviewer: Share free/busy hours, subject, location, and the body of the message or calendar item.
+    
+    Anonymous:ContactsSharing: Share contacts only.
```

Comment:
Share contact only doesn't allow viewing the Calendar  Freebusy status of the organizer, why added?